src/font-weight.scss

```diff
@@ -1,6 +1,9 @@
-:root, :host {
-  --font-weight-light: 300;
+:root,
+:host {
   --font-weight-regular: 400;
-  --font-weight-medium: 500;
   --font-weight-bold: 700;
+
+  @mixin font-style-italic() {
```

Comment:
just spotted that this mixin isn't being used, and since it's only the one style it could be a variable really, unless there's a future plan for it?